### PR TITLE
Structure data cleaning

### PR DESCRIPTION
**Pull request by Stefenon**
Pull request for the data cleaning stage. The code used in the script is justified on the notebook called '1.0-vse-data-cleaning-structure.ipynb'.

### CODE REVIEW

File: notebooks/1.1-vse-eda-rooms.ipynb [hunk 1]
```diff
@@ -1,943 +0,0 @@
-{
```
Comment (yvesfracari) [inline notebooks/1.1-vse-eda-rooms.ipynb:1]: This pull request is about data cleanning. So, this file shouldn't be here.

File: src/data/vse_structural_data_cleaning_script.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+train['OverallQual'].replace([1, 2, 3, 4], '4-', inplace=True)
```
Comment (yvesfracari) [inline src/data/vse_structural_data_cleaning_script.py:1]: I prefer to create a function and done all these changes in the dataframe inside it.

File: notebooks/1.1-vse-eda-rooms.ipynb (hunk 1 above)
Comment (Stefenon) [reply notebooks/1.1-vse-eda-rooms.ipynb:1]: Yeah, I realized it was there afterwards and deleted it lol

**Comment by Stefenon** [comment]
> The commits you made were much better that last time! Continue improving your git skills hahaha!

File: src/data/vse_structural_data_cleaning_script.py (hunk 2 above)
Comment (Stefenon) [reply src/data/vse_structural_data_cleaning_script.py:1]: Didn't understand what you mean. You mean creating a function that substitutes the replace method?

> Good job on the data cleaning. My principal feedback is for you add some comments about what you are doing. Put commens like "how it was", "what Im gonna do to change it". In this way other people can understand better.

Ok! I'm gonna try to do this more often and more clearly next time.

> > The commits you made were much better that last time! Continue improving your git skills hahaha!

Thanks!

**Comment by Stefenon** [comment]
> The commits you made were much better that last time! Continue improving your git skills hahaha!

Thanks!

**Comment by Stefenon** [comment]
> Good job on the data cleaning. My principal feedback is for you add some comments about what you are doing. Put commens like "how it was", "what Im gonna do to change it". In this way other people can understand better.

Ok! I will try to do this more often and more clearly.

**Pull request closed without merging**